utils/installer/lv-config.example.lua

```diff
@@ -82,6 +82,13 @@ O.lang.go.autoformat = true
 
 -- rust
 O.lang.rust.autoformat = true
+
+-- clang
+O.lang.clang.active = false -- make this true to enable clang support
+O.lang.clang.autoformat = false -- make this true to enable auto complete in C, C++ files
```

Comment:
auto complete in -> auto formatting for ?

Reply:
sorry, if this caused some misunderstanding, by `auto formatting`. I meant indentation stuff on saving code. I had terrible time with `treesitter indentation` with `autopairs` as mentioned in #563 .